### PR TITLE
Updating the ktoml version to 0.2.11

### PR DESCRIPTION
**Pull request by orchestr7**
### What's done:
- updated ktoml version from 0.2.9 to 0.2.11
- changed tests to reflect changes in the API (exceptions)

Closes #13

### CODE REVIEW

**Comment by orchestr7** [comment]
Sorry for bugs and thank you for your report!

File: src/backendTest/kotlin/com/jakewharton/posterbox/ConfigTest.kt [hunk 1]
```diff
@@ -99,23 +95,23 @@ class ConfigTest {
...
 	@Test fun plexHostMissingThrows() {
-		val t = assertFailsWith<KtomlException> {
+		val t = assertFailsWith<TomlDecodingException> {
 			Config.parseFromToml("""
 			|[plex]
 			|token = "abc123"
 			|""".trimMargin())
 		}
-		assertTrue("Missing the required field <host>" in t.message!!)
+		assertContains(t.message!!, "Missing required property <host>")
```
Comment (orchestr7) [inline src/backendTest/kotlin/com/jakewharton/posterbox/ConfigTest.kt:106]: May be `assertContains` looks better?

Comment (JakeWharton) [reply src/backendTest/kotlin/com/jakewharton/posterbox/ConfigTest.kt:106]: Yes, thanks! I didn't know this existed.

**Comment by JakeWharton** [comment]
Thanks for the PR!

**Pull request merged**